LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 OVO Energy
+   Copyright 2018 OVO Energy
```

Comment:
Not 2021? lol

Reply:
the date is start date actually. I did this by mistake, will revert